### PR TITLE
nlohmann_json 3.11.2

### PR DESCRIPTION
**Pull request by skupr-anaconda**
**libmamba 1.4.1** requires it

Changelog: https://github.com/nlohmann/json/blob/develop/ChangeLog.md
License: https://github.com/nlohmann/json/blob/v3.11.2/LICENSE.MIT
Requirements: https://github.com/nlohmann/json/blob/v3.11.2/CMakeLists.txt

Actions:
1. Update test commands
2. Remove doc_source_url

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {% set name = "nlohmann_json" %}
-{% set version = "3.10.5" %}
-{% set sha256 = "5daca6ca216495edf89d167f808d1d03c4a4d929cef7da5e10f135ae1540c7e4" %}
+{% set version = "3.11.2" %}
+{% set sha256 = "d69f9deb6a75e2580465c6c4c5111b89c4dc2fa94e3a85fcd2ffcd9a143d9273" %}
 {% set build_number = "0" %}
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml]: variable not needed as used only once

File: recipe/meta.yaml [hunk 2]
```diff
@@ -9,7 +9,6 @@ package:
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml:12]: not needed

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {% set name = "nlohmann_json" %}
-{% set version = "3.10.5" %}
-{% set sha256 = "5daca6ca216495edf89d167f808d1d03c4a4d929cef7da5e10f135ae1540c7e4" %}
+{% set version = "3.11.2" %}
+{% set sha256 = "d69f9deb6a75e2580465c6c4c5111b89c4dc2fa94e3a85fcd2ffcd9a143d9273" %}
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml]: variable not needed as used only once

File: recipe/meta.yaml (hunk 1 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml]: Fixed

File: recipe/meta.yaml (hunk 2 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml:12]: Fixed

File: recipe/meta.yaml (hunk 3 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml]: Fixed

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

**Pull request merged**